modules/CmpdReg/src/client/custom/LotView_Custom.inc

```diff
@@ -70,7 +70,7 @@
 			</div>
 			<div>
 				<label class="FormLabel"><span class="notForVirtual">*</span>Synthesis Date:</label>
-				<input type="textfield" class="FormInput synthesisDate" <%- saved ? 'disabled=true': '' %> value="<%- synthesisDate %>"/>
+				<input type="textfield" class="FormInput synthesisDate" value="<%- synthesisDate %>"/>
```

Comment:
This line and the 2 above just remove the logic that said to disable the field if it was saved.  Instead, it's always editable as long as the lot itself is editable.